src/pmdas/linux/pmda.c

```diff
@@ -2006,6 +2008,36 @@ static pmdaMetric metrictab[] = {
     { PMDA_PMID(CLUSTER_NET_NFS,70), PM_TYPE_U32, PM_INDOM_NULL, PM_SEM_COUNTER,
     PMDA_PMUNITS(0,0,1,0,0,PM_COUNT_ONE) } },
 
+/* nfs.server.threads.total */
+  { &proc_fs_nfsd.th_cnt,
+    { PMDA_PMID(CLUSTER_NET_NFS,71), PM_TYPE_U32, PM_INDOM_NULL, PM_SEM_COUNTER,
+    PMDA_PMUNITS(0,0,1,0,0,PM_COUNT_ONE) } },
+
+/* nfs.server.threads.pools */
+  { &proc_fs_nfsd.pool_cnt,
+    { PMDA_PMID(CLUSTER_NET_NFS,72), PM_TYPE_U32, PM_INDOM_NULL, PM_SEM_COUNTER,
+    PMDA_PMUNITS(0,0,1,0,0,PM_COUNT_ONE) } },
+
+/* nfs.server.threads.requests */
+  { &proc_fs_nfsd.pkts_arrived,
+    { PMDA_PMID(CLUSTER_NET_NFS,73), PM_TYPE_U32, PM_INDOM_NULL, PM_SEM_COUNTER,
```

Comment:
Checking the kernel source, svc_xprt.c ~ svc_pool_stats_show some of these look like unsigned longs.  In pcp we have KERNEL_ULONG instead of the PM_TYPE_* to deal with that.  Mind changing this one as well as sockets-enqueued, threads_woken and threads_timeout?